models/siegert_neuron.h

```diff
@@ -73,21 +75,21 @@ The following parameters can be set in the status dictionary.
 =====  ====== ==============================
  rate  1/s    Rate (1/s)
  tau   ms     Time constant
- mean  real   Additional constant input
+ mean  1/s    Additional constant input
 =====  ====== ==============================
 
 The following parameters can be set in the status directory and are
 used in the evaluation of the gain function. Parameters as in
 iaf_psc_exp/delta.
 
-
-=========  ======  =====================================================
+\verbatim embed:rst
```

Comment:
@sarakonradi: is this statement necessary after #1530?

Reply:
@clinssen It is not. In fact, this needs to be removed. Otherwise the table won't render properly!